### PR TITLE
Improvement - Updated references, Banner landingpage placement, readme update

### PR DESCRIPTION
**Pull request by mmexvr**
**Describe pull-request**  
_Describe what the pull-request is about_

- Fixing issue and PR templates
- Fixing reference to old corporate-ui
- Updated readme
- Try and download pictogram(Linking to new repo now for the download)
- Landing page had wrong columns on grid(Might need to remove the fluid talk with niklas regarding using fluid or not)
- Caption color
- Package.json
- Remove duplicate css from landing page

**Solving issue**  
_Add which issue this pull-request solves by adding # plus the number of the issue (for example #123)_
Fixes: #201 
Fixes: #165 
Also, but needs to discusses, fixes: #182

**How to test**  
_Add description how to test if possible_
- Read the readme.md
- Read issues
- Look at site with image caption, correct color
- Look at the banner in bottom, related with #182 
- Packages json

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,39 +20,43 @@ Clone this repo, install all dependencies and start the application:
...
 ```bash
-> cd corporate-ui-site
+> cd sdds-website
 > npm run data
 ```
 
-## Developing with Corporate-Ui and Scania-Theme
+## Running [@scania-sdds/components](https://www.npmjs.com/package/@scania-sdds/components) and [@scania-sdds/theme-light](https://www.npmjs.com/package/@scania-sdds/theme-light) on sdds-website
+
+This is used to be able to run the local version on your compunter toghether with the website. To find out more about the SDDS component and themes packages please visit [scania-digital-design-system/sdds](https://github.com/scania-digital-design-system/sdds)
```
Comment (helloimela) [inline README.md:41]: Typo: computer

**Review by helloimela — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,20 +1,17 @@
...
 
-Visit our Design System at [Digitaldesign.scania.com](https://digitaldesign.scania.com/)
-
-The application uses **SDDS** and the base structure is **Angular 8**.
-
-## How to get started with Corporate-Ui and Scania-theme
+Visit SDDS at [digitaldesign.scania.com](https://digitaldesign.scania.com/)
+## How to get started with @scania-sdds/components and @scania-sdds/theme-light
 
 For more information how to contribute [Digitaldesign.scania.com/getting-started/](https://digitaldesign.scania.com/getting-started/development)
```
Comment (helloimela) [inline README.md:6]: How to get started using it and how to contribute is two different thing.
- Under this title, can change the sentence to: Check the documentation in [link]
- Can create a separate section for "How to contribute" and it should link to https://github.com/scania-digital-design-system/sdds/blob/master/CONTRIBUTING.md

Comment (mmexvr) [reply README.md:6]: Should probably link also to the contribution page

**Review by helloimela — APPROVED** [review]

**Review by helloimela — APPROVED** [review]

**Review by niklasros — APPROVED** [review]

**Review by helloimela — APPROVED** [review]
Great job!

**Pull request merged**